test.py

```diff
@@ -8,6 +8,7 @@
 from wordvecspace import annoy
 from wordvecspace import disk
 
+
```

Comment:
I think extra line space is not required, though auto pep8 by default includes 2 line space, it looks odd to me. Please check